python/ray/tune/examples/mnist_pytorch_lightning.py

```diff
@@ -259,7 +262,8 @@ def tune_mnist_pbt(num_samples=10, num_epochs=10, gpus_per_trial=0):
             train_mnist_tune_checkpoint,
             data_dir=data_dir,
             num_epochs=num_epochs,
-            num_gpus=gpus_per_trial),
+            # Handle fractional GPU case. Training func should still take int.
+            num_gpus=math.ceil(gpus_per_trial)),
```

Comment:
can we put this in `Trainer(gpus=math.ceil(...))` instead of here?

Reply:
Good point, updated.